### PR TITLE
Fix a bug in positionFixed where parent is transformed + some code clean up

### PR DESCRIPTION
**Pull request by MrFrankel**
positionFixed has been running in our production app for a while now with no issues other then this one.
Browser calculate position: fixed relative to viewport unless a parent has a transform property defined.

I have added a fix for this with tests included for it.

Also, I would like to add positionFixed to the documents in order for other to be able to make use of it, but I dont know how.

### CODE REVIEW

File: packages/popper/src/utils/isIE.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/**
```
Comment (FezVrasta) [inline packages/popper/src/utils/isIE.js]: 1. Do we really need to check of IE11 as well? If not, can we use the existing `isIE10` function?

**Comment by FezVrasta** [comment]
Documentation is automatically generated from the JSDocs comments

**Comment by MrFrankel** [comment]
yes we do, apparently the transform ancestor thing is on all browsers except < ie11.

we dont actually support ie anymore, it is an absolute pain!

**Comment by MrFrankel** [comment]
ok, missed the second bullet, sure, ill make IE more generic, good point

**Comment by MrFrankel** [comment]
@FezVrasta made the isIE more generic, anything else you see needs addressing?

**Pull request merged**